subscribe.go

```diff
@@ -50,6 +50,8 @@ type HandlerOptions struct {
 	// StartFromBeginning starts a new subscriber from
 	// the beginning of messages available, if supported
 	StartFromBeginning bool
+	// Unique subscriber means that all subscribers will receive all messages
```

Comment:
Maybe this needs a bit more clarification. As `Unique` is not really useful if there's just one process pulling from each subscription.